cafe_information

```diff
@@ -0,0 +1,21 @@
+create table CafeInSanFrancisco (
+	name varchar(32) NOT NULL,
+	address varchar(64) NOT NULL,
+	latitude double(18,14) NOT NULL,
+	longitude double(18,14) NOT NULL,
+	rating float NOT NULL,
+	Monday char(11) NOT NULL,
```

Comment:
What does Monday to Sunday mean? This structure seems too complicated. I think we can merge these into one column.

Reply:
I think we can reduce all week-related columns below.

Reply:
@harrydrippin Just out of curiosity, if we combine these into one column, will it contain one huge string that needs parsing later on? e.g. Monday (open from 11am to 5pm), Tuesday (closed) , Wednesday (open from 9am to 6pm), Thursday (open from 10am to 5pm), Friday (open from 9am to 3pm), Saturday (closed), Sunday (open from noon to 4pm)

Reply:
I think it doesn't need to combine into one. It seems easier to parse.

Reply:
I'm the same opinion with @shberry and @khc196, we need the split data.

Reply:
@harrydrippin then, should I write in lowercase for Monday?